src/main/java/com/tieumy/exercise/ArrayEx10.java

```diff
@@ -0,0 +1,15 @@
+package com.tieumy.exercise;
+
+public class ArrayEx10 {
+
+    public int[] reverseArray(final int[] arr) {
+        int length = arr.length;
+        int[] reversedArray = new int[length];
+        int j = length;
+        for (int i = 0; i < length; i++) {
+            reversedArray[j - 1] = arr[i];
+            j--;
+        }
```

Comment:
Có thể để j làm biến vòng for

```suggestion
        for (int i = 0, j = length - 1; i < length; i++, j--) {
            reversedArray[j] = arr[i];
        }
```